packages/node_modules/nav-frontend-alertstriper/src/alertstripe.js

```diff
@@ -28,29 +29,31 @@ const ikonKind = (type, solid) => {
 
 class AlertStripe extends Component { // eslint-disable-line react/prefer-stateless-function
     render() {
-        const { type, solid, className, children, ...props } = this.props;
+        const { type, solid, size, className, children, ...props } = this.props;
 
         return (
             <div className={cls(type, solid, className)} {...props}>
                 <span className="alertstripe__ikon">
-                    <Ikon kind={ikonKind(type, solid)} height="1.5rem" width="1.5rem" />
+                    <Ikon kind={ikonKind(type, solid)} size={size} />
                 </span>
-                <Normaltekst tag="span">{children}</Normaltekst>
+                <Normaltekst className="alertstripe__tekst" tag="span">{children}</Normaltekst>
             </div>
         );
     }
 }
 
 AlertStripe.propTypes = {
     children: PT.node.isRequired,
-    type: PT.oneOf(['advarsel', 'suksess', 'info']).isRequired,
+    type: PT.oneOf(['advarsel', 'suksess', 'info', 'nav-ansatt']).isRequired,
     solid: PT.bool,
+    size: PT.string,
```

Comment:
Size-property'en tenker jeg egentlig at vi burde ha en custom validator på som gir warning for tilfellet props.type !== 'nav-ansatt' && props.size !== AlertStripe.defaultProps.size. De andre tre typene av AlertStripe er forutbestemte hva gjelder størrelse på ikonene og på alertstripen.